### PR TITLE
Add Sui to glossary and note its use for currency

### PR DESCRIPTION
**Pull request by Clay-Mysten**

### CODE REVIEW

File: doc/src/learn/sui-glossary.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ stage is considered closure for an exchange or other blockchain transaction.
 
 ### Gas
 
-As with other blockchains, [gas](https://www.investopedia.com/terms/g/gas-ethereum.asp) in Sui is the currency for the cost of conducting a transaction.
+As with other blockchains, [gas](https://www.investopedia.com/terms/g/gas-ethereum.asp) refers to the currency for the cost of conducting a transaction. In Sui, the currency is eponymous: Sui
```
Comment (666lcz) [inline doc/src/learn/sui-glossary.md:70]: Should we refer the currency as `SUI` instead of `Sui`?

Comment (huitseeker) [reply doc/src/learn/sui-glossary.md:70]: Yep, this is the industry convention: ticker caps for coins.

Comment (Clay-Mysten) [reply doc/src/learn/sui-glossary.md]: Thank you both. I fixed this here and added in in the separate Sui definition.

**Review by bwann52 — CHANGES_REQUESTED** [review]
Gas is the cost of transaction but not the currency.

Feedback from Alonso:

i think we should follow existing conventions and i) refer to gas as gas, ii) refer to the unit cost of gas in our native currency SUI as the gas price, iii) refer the product of gas * gas_price as the transaction fee

**Comment by Clay-Mysten** [comment]
Asking Alonso to take another look. Thank you!

**Comment by econmysten** [comment]
I don't this entry is accurate.
1) I'd rather include a link from the Ethereum foundation than from investopedia: https://ethereum.org/en/developers/docs/gas/
2) I prefer the wording in the former.
3) I propose: "Gas refers to the computational effort required for executing operations on the Sui network. In Sui, gas is paid with the network's native currency SUI. The cost of executing a transaction in SUI units is referred to as the transaction fee."

**Review by econmysten — APPROVED** [review]

**Comment by Clay-Mysten** [comment]
> Gas is the cost of transaction but not the currency.
> 
> Feedback from Alonso:
> 
> i think we should follow existing conventions and i) refer to gas as gas, ii) refer to the unit cost of gas in our native currency SUI as the gas price, iii) refer the product of gas * gas_price as the transaction fee

This should be all done. Please take another look, Bauer. Thanks!

**Pull request merged**